games-fps/redeclipse/redeclipse-1.6.0-r2.ebuild

```diff
@@ -78,6 +78,5 @@ src_install() {
 	dodoc readme.txt doc/examples/servinit.cfg
 
 	dobin "${FILESDIR}/redeclipse"
-	cd /usr/bin || die
-	dosym redeclipse redeclipse_server
+	dosym "${EPREFIX}redeclipse" /usr/bin/redeclipse_server
```

Comment:
Now you're linking it to `/home/mygentooredeclipse`.

Reply:
Weird, because I manually checked the symlink immediately after emerging, and it was correct. Only reason I was trying to figure out the usage of `${EPREFIX}` is because of repoman. I used `qgrep` to see how many other ebuilds issue `dosym` and switched to that format as such:

`dosym redeclipse /usr/bin/redeclipse`

Again, manually inspected, symlink is set correctly after emerge.

Reply:
That's because you aren't using Gentoo Prefix, and so `${EPREFIX}` is empty. It wouldn't work if it was non-empty.